float64 gpst
float64 lat
float64 lon
float32 h
float32[3] pos_std
uint8 mode
uint8 ns